port/raspberrypi/rp2xxx/src/hal/usb.zig

```diff
@@ -360,8 +360,8 @@ pub fn F(comptime config: UsbConfig) type {
             var setup_packet: [8]u8 = .{0} ** 8;
             const spl: u32 = peripherals.USB_DPRAM.SETUP_PACKET_LOW.raw;
             const sph: u32 = peripherals.USB_DPRAM.SETUP_PACKET_HIGH.raw;
-            _ = rom.memcpy(setup_packet[0..4], std.mem.asBytes(&spl));
-            _ = rom.memcpy(setup_packet[4..8], std.mem.asBytes(&sph));
+            @memcpy(setup_packet[0..4], std.mem.asBytes(&spl));
```

Comment:
Does this crash the chip per the comment in `rom.zig`?

Reply:
This part don't crash chip because they are aligned, so I could remove this changes from PR.

But this will crash, when changed to `@memcpy`:
```
_ = rom.memcpy(ep.data_buffer[0..buffer.len], buffer);
```

Reply:
I think new zig 0.14.0 should work fine with `@memcpy` https://github.com/ziglang/zig/blob/183bb8b08403ab594cdc543c2db2503e7eac0052/lib/compiler_rt/memcpy.zig#L4
now they have code with alignment 🤔

Reply:
Ahhh I understand now, the rom function doesn't obey alignment. All good!